src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -156,6 +156,11 @@ Config file must be valid JSON format. Examples can be found under the test fold
         value: true | false
         if not supplied: false
 
+      "onlyOneWinnerPerRound" optional
+        in a multi-seat contest, elect no more than one winner per round, even when there are multiple candidates exceeding the winning threshold
```

Comment:
This makes me think we should add a validation rule that checks to make sure it's not on in a non-multi-seat election.

Reply:
The validation feedback on existing rules was quite helpful when I was testing and trying to break things :)

Reply:
I thought about doing that, but I wasn't sure if it made sense. Setting this to true isn't actually incompatible with a single-seat contest; it's just irrelevant.